tests/flow_test.go

```diff
@@ -523,17 +495,12 @@ func TestFlowGremlin(t *testing.T) {
 }
 
 func queryFlowMetrics(gh *gclient.GremlinQueryHelper, bridge string, timeContext int64, pings int64) error {
-	graphGremlin := "g"
-	if timeContext != -1 {
-		graphGremlin += fmt.Sprintf(".Context(%d)", timeContext)
-	}
-
-	ovsGremlin := graphGremlin + fmt.Sprintf(`.V().Has("Name", "%s", "Type", "ovsbridge")`, bridge)
+	ovsGremlin := g.G.Context(timeContext).V().Has("Name", bridge, "Type", "ovsbridge")
 	if _, err := gh.GetNode(ovsGremlin); err != nil {
 		return err
 	}
 
-	gremlin := ovsGremlin + `.Flows().Has("LayersPath", Regex(".*ICMPv4.*"))`
+	gremlin := ovsGremlin.Flows().Has("LayersPath", g.Contains("ICMPv4"))
```

Comment:
I would prefer to keep the query builder as close to the one implemented by Skydive. Why not g.Regex(".*ICMPv4.*")